metasrv/src/executor/meta_handlers.rs

```diff
@@ -143,6 +146,35 @@ impl RequestHandler<DropDatabaseReq> for ActionHandler {
     }
 }
 
+#[async_trait::async_trait]
+impl RequestHandler<RenameDatabaseReq> for ActionHandler {
```

Comment:
This seems not a KV API. Is it the old-way meta API?

Reply:
> This seems not a KV API. Is it the old-way meta API?

After all meta API has impl by KV api, all the meta handlers in old way will be removed. But now, in order to pass compile, it has to impl with an old way API.